tgui/public/tgui.bundle.js

```diff
@@ -1 +1 @@
ar n=t.props,r=n.onInput,o=n.onChange,i=n.onEnter;return e.keyCode===a.KEY_ENTER?(t.setEditing(!1),o&&o(e,e.target.value),r&&r(e,e.target.value),i&&i(e,e.target.value),void(t.props.selfClear?e.target.value="":e.target.blur())):e.keyCode===a.KEY_ESCAPE?t.props.onEscape?void t.props.onEscape(e):(t.setEditing(!1),e.target.value=d(t.props.value),void e.target.blur()):void 0},t}n=e,(t=f).prototype=Object.create(n.prototype),t.prototype.constructor=t,l(t,n);var p=f.prototype;return p.componentDidMount=function(){var e=this,t=this.props.value,n=this.inputRef.current;n&&(n.value=d(t)),(this.props.autoFocus||this.props.autoSelect)&&setTimeout((function(){n.focus(),e.props.autoSelect&&n.select()}),1)},p.componentDidUpdate=function(e,t){var n=this.state.editing,r=e.value,o=this.props.value,i=this.inputRef.current;i&&!n&&r!==o&&(i.value=d(o))},p.setEditing=function(e){this.setState({editing:e})},p.render=function(){var e=this.props,t=(e.selfClear,e.onInput,e.onChange,e.onEnter,e.value,e.maxLength),n=e.placeholder,a=s(e,c),l=a.className,d=a.fluid,f=a.monospace,p=s(a,u);return(0,r.normalizeProps)((0,r.createComponentVNode)(2,i.Box,Object.assign({className:(0,o.classes)(["Input",d&&"Input--fluid",f&&"Input--monospace",l])},p,{children:[(0,r.createVNode)(1,"div","Input__baseline",".",16),(0,r.createVNode)(64,"input","Input__input",null,1,{placeholder:n,onInput:this.handleInput,onFocus:this.handleFocus,onBlur:this.handleBlur,onKeyDown:this.handleKeyDown,maxLength:t},null,this.inputRef)]})))},f}(r.Component);t.Input=f},61515:function(e,t,n){"use strict";t.__esModule=!0,t.KeyListener=void 0;var r=n(96323),o=n(6965);function i(e,t){return(i=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e})(e,t)}var a=function(e){var t,n;function r(){var t;return(t=e.call(this)||this).dispose=void 0,t.dispose=(0,o.listenForKeyEvents)((function(e){t.props.onKey&&t.props.onKey(e),e.isDown()&&t.props.onKeyDown&&t.props.onKeyDown(e),e.isUp()&&t.props.onKeyUp&&t.props.onKeyUp(e)})),t}n=e,(t=r).prototype=Object.create(n.prototype),t.prototype.constructor=t,i(t,n);var a=r.prototype;return a.componentWillUnmount=function(){this.dispose()},a.render=function(){return null},r}(r.Component);t.KeyListener=a},14274:function(e,t,n){"use strict";t.__esModule=!0,t.Knob=void 0;var r=n(96323),o=n(97622),i=n(1060),a=n(54425),c=n(42833),u=n(97667),s=["animated","format","maxValue","minValue","unclamped","onChange","onDrag","step","stepPixelSize","suppressFlicker","unit","value","className","style","fillValue","color","ranges","size","bipolar","children"];t.Knob=function(e){if(Byond.IS_LTE_IE8)return(0,r.normalizeProps)((0,r.createComponentVNode)(2,u.NumberInput,Object.assign({},e)));var t=e.animated,n=e.format,l=e.maxValue,d=e.minValue,f=e.unclamped,p=e.onChange,h=e.onDrag,m=e.step,v=e.stepPixelSize,g=e.suppressFlicker,y=e.unit,b=e.value,_=e.className,w=e.style,C=e.fillValue,x=e.color,N=e.ranges,E=void 0===N?{}:N,S=e.size,V=void 0===S?1:S,k=e.bipolar,O=(e.children,function(e,t){if(null==e)ret
```

Comment:
I really hope this is right because I can't read the diff on this at all.

Reply:
It should be correct. I added a tgui interface while making this but since decided to get rid of it. I ran tgui-build again and this should be the correct bundle.